deploy-service/common/src/main/java/com/pinterest/deployservice/common/HTTPClient.java

```diff
@@ -31,7 +31,7 @@
     private static final int TIMEOUT = 15*1000;  // http timeout in 15 seconds
     private static final Logger LOG = LoggerFactory.getLogger(HTTPClient.class);
 
-    private String generateUrlAndQuery(String url, Map<String, String> params) throws Exception {
+    public String generateUrlAndQuery(String url, Map<String, String> params, boolean scrubUrl) throws Exception {
```

Comment:
Temporarily converted to public method to discuss testing practices (see unit tests file).